packages/cli/src/cmds/add.ts

```diff
@@ -226,7 +228,15 @@ export default class Add extends YargsCommand {
       ? [
           // Category 1: Add Teams Capability
           ...(isBotNotificationEnabled()
-            ? [new CapabilityAddNotification(), new CapabilityAddCommandAndResponse()]
+            ? [
+                ...(isWorkflowBotEnabled()
```

Comment:
Maybe only checking the feature flag for workflow bot is better. Leave notification and command bots out of this ternary operator.

Reply:
Updated. Thanks~